src/internet_identity/internet_identity.did

```diff
@@ -215,6 +215,17 @@ type IdentityAnchorInfo = record {
     device_registration: opt DeviceRegistrationInfo;
 };
 
+type GetDeviceCredentialsResponse = record {
```

Comment:
This is very descriptive, have you considered something like `Credentials` or `CredentialsInfo` or `AnchorCredentials`?